packages/components/pie-tag/src/tag.scss

```diff
@@ -1,5 +1,39 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
+@mixin tag-interactive-states($bg-color, $mode: 'default') {
+    &.c-tag--interactive {
+        &:hover:not(.c-tag--disabled) {
+            @if $mode == 'inverse' {
+                --hover-modifier: var(--dt-color-hover-02);
+            } @else {
+                --hover-modifier: calc(-1 * var(--dt-color-hover-01));
```

Comment:
Out of curiosity: what the multiplication to a negative number does to a CSS color?

Reply:
Good question. In this case it probably indicates poor naming of the variable, it's actually a percentage rather than a colour, it's like applying a black overlay with X% opacity.

Reply:
Thanks!